pkg/api/component_report.go

```diff
@@ -253,10 +215,6 @@ func (c *componentReportGenerator) getJobRunTestStatusFromBigQuery() (
 						AND @Variant IN UNNEST(variants)
 						AND cm.id = @TestId `
 	commonParams := []bigquery.QueryParameter{
-		{
-			Name:  "IgnoredJobs",
-			Value: ignoredJobsRegexp,
-		},
```

Comment:
In the context of a productized sippy, it might be best to keep the ignored jobs configurable. I'd expect we'll want view schema managed automatically and common for any deployment of sippy, but the jobs to ignore certain jobs something configured in the app itself and applied dynamically. WDYT.

Reply:
It doesn't quite work with the materialized view (scheduled query) to do it that way, because I'm precomputing test results for whole days grouped by network/upgrade/arch/platform/variants. I had planned to make the ignored jobs configurable in Sippy when it creates the views.

If I do it at the job level, it'll be less useful for performance/cost, but I'm not sure how much, I'll look into it